manual/manual/cmds/intf-c.etex

```diff
@@ -2023,12 +2023,13 @@ stub that ignores all the special annotations.
 \subsection{Passing unboxed values}
 
 We said earlier that all OCaml objects are represented by the C type
-"value", and one has to use macros such as "Int_val" to decode data
-from the "value" type. It is however possible to tell OCaml to do this
-for us and pass arguments unboxed to the C function. Similarly it is
-possible to tell OCaml to expect the result unboxed and box it for us.
+"value", and one has to use macros such as "Int_val" to decode data from
+the "value" type.  It is however possible to tell the OCaml native code
+compiler to do this for us and pass arguments unboxed to the C function.
+Similarly it is possible to tell OCaml to expect the result unboxed and box
```

Comment:
If you want to avoid the OCaml native-code compiler repetition, you could use `ocamlopt` rather than `OCaml` .